src/applications/verify/containers/VerifyApp.jsx

```diff
@@ -6,20 +6,47 @@ import LoginGovSVG from 'platform/user/authentication/components/LoginGovSVG';
 import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
 import recordEvent from 'platform/monitoring/record-event';
 
-import { verify } from 'platform/user/authentication/utilities';
 import { hasSession } from 'platform/user/profile/utilities';
 import SubmitSignInForm from 'platform/static-data/SubmitSignInForm';
+import { CSP_IDS } from 'platform/user/authentication/constants';
 import { focusElement } from '~/platform/utilities/ui';
+import { VerifyButton } from '../components/verifyButton';
 
 export class VerifyApp extends React.Component {
   constructor(props) {
     super(props);
 
+    const { MHV, MHV_VERBOSE, DS_LOGON, LOGIN_GOV, ID_ME } = CSP_IDS;
+
     this.signinMethodLabels = {
-      dslogon: 'DS Logon',
-      myhealthevet: 'My HealtheVet',
-      logingov: 'Login.gov',
+      [DS_LOGON]: 'DS Logon',
+      [LOGIN_GOV]: 'Login.gov',
+      [MHV]: 'My HealtheVet',
+      [MHV_VERBOSE]: 'My HealtheVet',
+      [ID_ME]: 'ID.me',
     };
```

Comment:
Let's scrap this and import the `SERVICE_PROVIDERS` constant in the authentication/constants file and grab the `label` key.